mycroft/configuration/mycroft.conf

```diff
@@ -540,10 +562,9 @@
   // Speech to Text parameters
   // Override: REMOTE
   "stt": {
-    // Engine.  Options: "mycroft", "google", "wit", "ibm", "kaldi", "bing",
-    //                   "houndify", "deepspeech_server", "govivace", "yandex"
+    // select a STT plugin as described in the respective readme
     "module": "ovos-stt-plugin-server",
-    "fallback_module": "ovos-stt-plugin-vosk",
```

Comment:
platform support concerns for all offline options, rpi3 support in particular proved cumbersome for end users because this default value would eat up all memory